### ISSUE TITLE
autotune blade(s)

### ISSUE THREAD

**Issue by bonze82** [siklon/cpuminer-gc3355#23]
still unable to autotune gridseed blades even though you are auto-tuning a blade in your readme:
./minerd --gc3355=/dev/ttyACM0,/dev/ttyACM1,/dev/ttyACM2,... --gc3355-autotune --gc3355-chips=40 --freq=825 --url=stratum+tcp://doge.ghash.io:3333 --userpass=user.pass
is this a per-system problem or is this a completely different problem...also is your dropbox file updated for the rpi and could that be the problem

**Comment by MisterFixx** [comment]
yes, please add autotuning for the G-Blades, autotuning is really useful and we'd really appriciate if you would!
